src/app/[locale]/_component/footer/footer.tsx

```diff
@@ -29,19 +29,19 @@ export default function Footer({
       links: [
```

Comment:
Is this related to this PR?

Reply:
This was a fix that was added with this PR. I wrote it in the "List of changes proposed in this pull request" section as the second feature. There is no task in jira for this.

Reply:
Is this the fix for the javascript security warnings?